frontend/common/src/components/UserProfile/ProfileSections/LanguageInformationSection.tsx

```diff
@@ -93,51 +96,18 @@ const LanguageInformationSection: React.FunctionComponent<{
           </span>
         </p>
       )}
-      {bilingualEvaluation === BilingualEvaluation.CompletedEnglish && (
-        <p>
-          {intl.formatMessage({
-            defaultMessage: "Completed an official GoC evaluation:",
-            description:
-              "Completed a government of canada abbreviation evaluation label and colon",
-          })}{" "}
-          <span data-h2-font-weight="b(700)">
-            {intl.formatMessage({
-              defaultMessage: "Yes, completed English evaluation",
-              description: "Completed an English language evaluation",
-            })}
-          </span>
-        </p>
-      )}
-      {bilingualEvaluation === BilingualEvaluation.CompletedFrench && (
-        <p>
-          {intl.formatMessage({
-            defaultMessage: "Completed an official GoC evaluation:",
-            description:
-              "Completed a government of canada abbreviation evaluation label and colon",
-          })}{" "}
-          <span data-h2-font-weight="b(700)">
-            {intl.formatMessage({
-              defaultMessage: "Yes, completed French evaluation",
-              description: "Completed a French language evaluation",
-            })}
-          </span>
-        </p>
-      )}
-      {bilingualEvaluation === BilingualEvaluation.NotCompleted && (
-        <p>
-          {intl.formatMessage({
-            defaultMessage: "Completed an official GoC evaluation:",
-            description:
-              "Completed a government of canada abbreviation evaluation label and colon",
-          })}{" "}
-          <span data-h2-font-weight="b(700)">
-            {intl.formatMessage({
-              defaultMessage: "No",
-              description: "No, did not completed a language evaluation",
-            })}
-          </span>
-        </p>
-      )}
+      <p>
+        {intl.formatMessage({
+          defaultMessage: "Completed an official GoC evaluation:",
+          description:
+            "Completed a government of canada abbreviation evaluation label and colon",
+        })}{" "}
+        <span data-h2-font-weight="b(700)">
+          {bilingualEvaluation
```

Comment:
I think maybe this changed messed up the profile summary?
![image](https://user-images.githubusercontent.com/8978655/185627701-02fda5e6-163d-4a6f-ad0f-a73350c86771.png)

Reply:
Great catch! I must have forgotten to test null state 😅 

Fixed with [fix lang eval null state on profile](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3695/commits/5925f0fae84a43fad82cebbc31519120a0a69115)

Reply:
Diff looks good again!